code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -28,10 +28,10 @@
 	var/printing = FALSE
 	var/static/list/pill_bottle_wrappers
 	var/static/list/bottle_styles
-	var/list/safe_chem_list = list("antihol", "charcoal", "epinephrine", "insulin", "teporone", "silver_sulfadiazine", "salbutamol",
-									"omnizine", "stimulants", "synaptizine", "potass_iodide", "oculine", "mannitol", "styptic_powder",
-									"spaceacillin", "salglu_solution", "sal_acid", "cryoxadone", "blood", "synthflesh", "hydrocodone",
-									"mitocholide", "rezadone", "menthol")
+	var/list/safe_chem_list = list("antihol", "charcoal", "epinephrine", "insulin", "teporone", "salbutamol", "omnizine",
+									"stimulants", "synaptizine", "potass_iodide", "oculine", "mannitol", "spaceacillin",
+									"salglu_solution", "sal_acid", "cryoxadone", "blood", "hydrocodone","mitocholide",
+									"rezadone", "menthol")
```

Comment:
Wrong edited file, the actual list is in `hypospray.dm` for hyposprays

Reply:
SEAN!!!!!!